user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Configuring_external_authentication_via_an_identity_provider_using_SAML.md

```diff
@@ -230,9 +227,16 @@ Once you have established a trust relationship between DataMiner (i.e. the servi
       ```xml
```

Comment:
To make sure this example is valid for all supported versions, it would be better to change this to two examples, one prior to the change, one from that DataMiner version onwards.